### PR TITLE
HIVE-22942 ptest alt

### PR DESCRIPTION
**Pull request by kgyrtkirk**

### CODE REVIEW

**Comment by kgyrtkirk** [comment]
x

**Comment by kgyrtkirk** [comment]
another comment

**Comment by kgyrtkirk** [comment]
.

File: ql/src/test/queries/clientpositive/schema_evol_par_vec_table_dictionary_encoding.q [hunk 1]
```diff
@@ -1,3 +1,5 @@
+--! qt:disabled:falky?!
```
Comment (jcamachor) [inline ql/src/test/queries/clientpositive/schema_evol_par_vec_table_dictionary_encoding.q:1]: Do all these disabled q files have corresponding JIRAs?

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+properties([
+    // max 5 build/branch/day
```
[the remark this answers is not in the document]
Comment (kgyrtkirk) [reply Jenkinsfile:20]: I haven't thinked about that - if it starts; it usually left to be finished. however: anyone will be able to cancel builds on the jenkins ui manually
Killing an in-flight build might not be the best - however all the builds will be waiting for the lock to start - I believe at that point a check could be than if there is any other pending run - and abort the actual build if there are any....

File: ql/src/test/queries/clientpositive/schema_evol_par_vec_table_dictionary_encoding.q (hunk 1 above)
Comment (kgyrtkirk) [reply ql/src/test/queries/clientpositive/schema_evol_par_vec_table_dictionary_encoding.q]: added jira refs to which already had one - and opened a few new jiras to have that

File: Jenkinsfile (hunk 2 above)
[the remark this answers is not in the document]
Comment (kgyrtkirk) [reply Jenkinsfile:20]: opened https://issues.apache.org/jira/browse/HIVE-23563 to not forget this

**Pull request closed without merging**